packages/common/src/interfaces/formExport.ts

```diff
@@ -0,0 +1,66 @@
+// Order determines the CSV column order
```

Comment:
This file can be in the api only. FE doesn't need it.

Reply:
moved